diff --git a/CMakeLists.txt b/CMakeLists.txt
index 68eaf52..019ccfc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -158,7 +158,7 @@ else()
     set(MIOPEN_USE_MIOPENGEMM 0)
 endif()
 
-option(Boost_USE_STATIC_LIBS "Use boost static libraries" ON)
+option(Boost_USE_STATIC_LIBS "Use boost static libraries" OFF)
 set(BOOST_COMPONENTS filesystem system)
 add_definitions(-DBOOST_ALL_NO_LIB=1)
 find_package(Boost REQUIRED COMPONENTS ${BOOST_COMPONENTS})
